lib/Slack.cs

```diff
@@ -43,7 +43,7 @@ static class Slack
                     .Json
                     .JsonConvert
                     .DeserializeObject<SlackUserList>(responseContent)
-                    .Members.First(member => member.Id == userId).Name;
+                    .Members.First(member => member.Id == userId.Split('|')[0])?.Name ?? "";
```

Comment:
I wonder if we could get this logic into SlackMessage.FindMentionedUsers. If we can, we can get it under test. So I can give it a whack, do you have the repro steps for this one?